Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
```

Comment:
This presumably needs to be reverted before merging.

Marking the PR as "request changes" to ensure this doesn't go in.